sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_message_backcompat.py

```diff
@@ -31,6 +31,7 @@ class Settler(Protocol):
         def settle_messages(
             self,
             delivery_id: Optional[int],
+            delivery_tag: Optional[str],
```

Comment:
this is string but it is being based into settle_messages() (I assume for uamqp?) but in pyamqp this accepts bytes - is the difference in type expected ?

Reply:
good catch :)